test/unit/annotation_layer_spec.js

```diff
@@ -581,6 +581,97 @@ describe('Annotation layer', function() {
     });
   });
 
+  describe('CheckboxWidgetAnnotation', function() {
+    var checkboxWidgetDict;
+
+    beforeEach(function (done) {
+      checkboxWidgetDict = new Dict();
+      checkboxWidgetDict.set('Type', Name.get('Annot'));
+      checkboxWidgetDict.set('Subtype', Name.get('Widget'));
+      checkboxWidgetDict.set('FT', Name.get('Btn'));
+
+      done();
+    });
+
+    afterEach(function () {
+      checkboxWidgetDict = null;
+    });
+
+    it('should have proper flags',
+        function() {
```

Comment:
For all occurrences in this file: move this to the previous line. We only use this if it doesn't fit on one line anymore.